bagitobjecttransfer/bagitobjecttransfer/settings/base.py

```diff
@@ -1,149 +1,146 @@
 import os
 
-from decouple import config
+from decouple import Csv, config
+from recordtransfer.configuration import AcceptedFileTypes
```

Comment:
The configuration file should be moved to `bagitobjecttransfer/`. This file should not depend on `recordtransfer`